src/components/services/CopyToClipboard.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <Generic :item="item">
+    <template #indicator>
+      <div class="status"><i ref="copyIcon" class="fa-regular fa-copy fa-xl" :class="{'scale': animate}" @click="copy()"></i></div>

```

Comment:
I believe you can use `@animationend` instead of the `ref` + `mounted` function. 
```suggestion
      <div class="status"><i @animationend="animate=false" class="fa-regular fa-copy fa-xl" :class="{'scale': animate}" @click="copy()"></i></div>

```

Reply:
True. Removed.